AirCasting/Graph/Graph.swift

```diff
@@ -53,21 +54,29 @@ struct Graph: UIViewRepresentable {
     }
     
     func updateUIView(_ uiView: AirCastingGraph, context: Context) {
-        
         try? uiView.updateWithThreshold(thresholdValues: thresholds.rawThresholdsBinding.wrappedValue)
-        
+  
         let entries = stream.allMeasurements?.compactMap({ measurement -> ChartDataEntry? in
             let timeInterval = Double(measurement.time.timeIntervalSince1970)
             let chartDataEntry = ChartDataEntry(x: timeInterval, y: measurement.value)
             return chartDataEntry
         }) ?? []
         let allLimitLines = getLimitLines()
-        uiView.limitLines = allLimitLines        
+        uiView.limitLines = allLimitLines
         simplifyGraphline(entries: entries, uiView: uiView)
     }
     
     private func simplifyGraphline(entries: [ChartDataEntry], uiView: AirCastingGraph) {
-        if entries.count > simplifiedGraphEntryThreshold {
+        
+        var counter = 0
+        let startTime = uiView.lineChartView.lowestVisibleX
+        let endTime = uiView.lineChartView.highestVisibleX
+        
+        entries.forEach { entry in
+            (entry.x >= startTime && entry.x <= endTime) ? counter += 1 : nil
+        }
```

Comment:
What do you think about replacing this with: `let counter = entries.filter { $0.x >= startTime && $0.x <= endTime}.count` ?